hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -82,7 +84,7 @@
    * @See #CONNECTION_LOCK.
    */
   private static Connection connection = null;
-  private static final Object CONNECTION_LOCK = new Object();
+  //private static final Object CONNECTION_LOCK = new Object();
```

Comment:
Just delete the line. Don't comment out.

Reply:
doh, thanks.